common/src/Model/BrushFace.cpp

```diff
@@ -358,9 +360,10 @@ namespace TrenchBroom {
                 swap(m_points[1], m_points[2]);
             }
 
-            setPoints(m_points[0], m_points[1], m_points[2]);
-
-            m_texCoordSystem->transform(oldBoundary, m_boundary, transform, m_attributes, textureSize(), lockTexture, invariant);
+            const auto setPointsResult = setPoints(m_points[0], m_points[1], m_points[2]);
+            return kdl::map_result([&]() {
```

Comment:
`map_result` is named according to its signature, which maps a result to a result (with a potentially different value type) using a lambda. But when we change the API to use members, we can also rename this to `and_then`, like here: https://github.com/TartanLlama/expected/blob/master/README.md

Reply:
`and_then` could also be used for void results, which I would prefer. I don't want different names for very similary functions, also because we might want to treat results generically.

Reply:
I'll make these changes in a later PR: https://github.com/kduske/TrenchBroom/issues/2983#issuecomment-658147940